hs_docker_base

```diff
@@ -1 +1 @@
-Subproject commit 57202cf091564322236b10e354d03801dc2158d8
+Subproject commit 0b178b09012f38773a300bc290d4894074d4b3bc
```

Comment:
Should be: `commit 57202cf091564322236b10e354d03801dc2158d8` which refers to 1.6.10

Not sure why/how you have `commit 0b178b09012f38773a300bc290d4894074d4b3bc` as that refers to 1.6.8.

So yes, check out the correct SHA for hs_docker_base and commit that, which should then make all of this go away.